salvo-remote/sandboxes/README.md

```diff
@@ -0,0 +1,72 @@
+# Sandboxes for remote Salvo executions
+
+This directory contains definition of sandboxes. A sandbox are all the
+resources deployed in AWS for a single Salvo execution. At the minimum a sandbox will contain:
+
+- a VM running Nighthawk (the load generator).
+- a VM running Envoy (the system under test or SUT).
+- a VM running the Nighthawk test server (the test backend).
+
+Note that the subnets used between the components are part of the common
+infrastructure and are already deployed as part of the [VPC
+configuration](https://github.com/envoyproxy/ci-infra/blob/main/salvo-infra/vpc.tf)
+done in the
+[ci-infra](https://github.com/envoyproxy/ci-infra/tree/main/salvo-infra)
+repository.
+
+Various sandbox types differ in the amount of instances deployed, amount of
+support systems deployed and the topology.
+
+# Available Sandbox types
+
+## The default sandbox
+
+The default contains the bare minimum of components needed to execute Salvo
+tests. The diagram below outlines the topology and content of the default
+sandbox. The default sandbox is currently available for the `x64` architecture
+only.
+
+![Diagram 1 - the default sandbox](images/the_default_sandbox.png)
+
+# Sandbox creation
+
+When Salvo executes, the CI pipeline will produce binaries and VM disk images
+for all the components. Once that is done, a new CI job is executed in the
+`salvo-control` agent pool on AZP. This job is then picked up by a Salvo
+control VM that instruments the sandbox creation.
```

Comment:
Is the salvo control VM the same thing as 'Salvo Controller' in the Diagram?

Reply:
Thank you for pointing this out, adjusted to use common terminology.

Done in a4025efbcc8a4fff046b98bd54d718a517c1583f.